content/concept-xrp-ledger-intro.md

```diff
@@ -0,0 +1,88 @@
+# XRP Ledger Introduction
+
+The **XRP Ledger** is a decentralized cryptocurrency network. Like all such networks, it's powered by a peer-to-peer server and it enables anyone to send and receive value within the rules of the network. The XRP Ledger is the home of **XRP**, a cryptocurrency which was designed by its creators to bridge the many different currencies preferred by people worldwide. Ripple, the company that stewards development of the XRP Ledger, expects XRP to be a key part of enabling the "Internet of Value": a world in which money moves the way information does today.
+
+## The Best Digital Asset
+
+XRP is the native cryptocurrency of the XRP Ledger. Anyone with a cryptographic key and an internet connection can receive, hold, and send XRP to anyone else. From the beginning, XRP's creators have developed it to be a desirable bridge currency which can facilitate trades in any other currency. XRP has many properties which make it an appealing asset for many other use cases, too:
+
+- **[Censorship-resistant transaction processing][]:** No single party decides which XRP transactions are approved or not, and no one can "roll back" a transaction after it has happened. As long as those who choose to participate in the network keep it healthy, XRP can be sent and received in seconds.
```

Comment:
micro-nit 😂 : initial cap the text in bold - to match the init-cap case pattern used for the others?